test/fixtures/payloads/check_run_success.json

```diff
@@ -0,0 +1,317 @@
+{
+  "action": "created",
+  "check_run": {
```

Comment:
This is just GitHub's example from https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#check_run. Its structure matches "real" webhooks.